Makefile

```
@@ -0,0 +1,49 @@
+ARTIFACT_ID=k8s-loki
+MAKEFILES_VERSION=8.4.0
+VERSION=2.9.1-1
```

```suggestion
VERSION=5.23.0-0
```
start with a "-0", so that the first release has the "-1"

---

But this should be the tool version, not the official Chart version, right? So `2.9.1-0` I think.